canonical-web-and-design/ubuntu.com
# CUBE: refactor code and add meta data

## Done

- Fix typo in nav
- Revert to using `getAttribute` not `nextElementSibling` to be more specific when targeting share menu
- Make the `.is-faded` styling a placeholder which is then extended instead of extending a class
- Add meta descriptions 
- Add `js-` prefix to class

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube and http://0.0.0.0:8001/cube/microcerts
- Check (update?) meta descriptions and titles
- Check points raised in this this [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/8990) have been addressed
- Check refactored code

## Issue / Card

Fixes #9308
